model/message_export_post.go

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package model
+
+type MessageExport struct {
```

Comment:
I wanted to separate each of the major categories of information (Team, Channel, User, Post, ChannelMember) into sub-structs, but I couldn't get Gorp to reliably populate the sub-structs, so I gave up and made this object flat.